src/App/Command/Email/SendAdminNotificationEmailCommand.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Command\Email;
+
+class SendAdminNotificationEmailCommand extends AbstractSendEmailCommand
+{
+    /**
+     * @var string
+     */
+    protected $userName;
+
+    public function __construct(string $recipient, string $userName, string $userOrganization)
+    {
+        parent::__construct($recipient);
+        $this->userName = $userName;
+        $this->userOrganization = $userOrganization;
```

Comment:
`$this->userOrganization` should be defined as a class property (like `$this->userName`).

Reply:
Added it as a class property.